src/components/AvatarWithDisplayName.js

```diff
@@ -88,6 +92,22 @@ function AvatarWithDisplayName(props) {
                             textStyles={[props.isAnonymous ? styles.headerAnonymousFooter : styles.headerText, styles.pre]}
                             shouldUseFullTitle={isExpenseReport || props.isAnonymous}
                         />
+                        {!_.isEmpty(parentNavigationSubtitle) && (
```

Comment:
Can you share steps to get the AnonymousReportFooter? I don't know it off the top of my head and if you have the steps it would save me some time to check this 🙂

Reply:
Cool, i have an errand for the next 90 minutes, so once I'm back I'll check your comments. Meanwhile, if you find any other issues let me know 👍

Reply:
tested this and yes having it in AnonymousReportFooter looks fine to me for now.